charts/argo/templates/server-sa.yaml

```diff
@@ -3,4 +3,6 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ .Values.server.serviceAccount }}
+  annotations:
+{{ toYaml .Values.server.serviceAccountAnnotation | indent 4 }}
```

Comment:
What about using `servrer.serviceAccount.annotations`?

Reply:
Same comment as below for `controller`